spec.js

```diff
@@ -23,17 +25,50 @@ describe('Choko - Create account', () => {
 });
 
 describe('Choko - Blog', () => {
-  it('try to create blog post without permission', () => {
-    const ramdonName = faker.lorem.word();
-    const randomTitle = faker.lorem.words();
-    const randomText = faker.lorem.sentences();
+  // instantiating a new instance of CreateBlogPost
+  const createBlogPost = new CreateBlogPost();
+
+  // defining references to faker and exposing those throughout the "Choko - Blog" describe block
```

Comment:
Do you really think this is a necessary comment? I always prefer readable code than code with comments.